testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -289,6 +289,44 @@
        and contains(@class,'node')]/ul/li/div/a[contains(.,'System List')]").click
 end
 
+#
+# Menu links
+#
+
+# Menu link: 1st level
+When(/^I follow the left menu "([^"]*)"$/) do |menu_level1|
+  xpath_menu_level1 = "//aside/div[@id='nav']/nav/ul/li/div/a[contains(.,'#{menu_level1}')]" # reach the link
+  find(:xpath, xpath_menu_level1).click
```

Comment:
"find" method will be critical, it will be call tons of times.
We should be sure we have it optimised, probably is just fine like that, but I think "find" inspect the whole page, maybe better to use "first". Something to test and compare ;)

Reply:
Right.